### PR TITLE
Loading image added.

### LINKED ISSUE #26: Loading screen on schedule page
Schedule page is loading data via Ajax, leading to a noticeable delay (during which the page is just blank) - we should add a small loader animation instead.

### PR DESCRIPTION
**Pull request by SilentFlame**
resolves #26 
@joerx have a look, if that's what is needed,

load
 you close the Issue and merge the PR.

### CODE REVIEW

File: schedule/index.html [hunk 1]
```diff
@@ -120,12 +123,17 @@ <h2 class="track-title">$track-title</h2>
...
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"
+      integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS"
       crossorigin="anonymous"></script>
     <script type="text/javascript">
       var ts = new Date().valueOf() // cache breaker
       document.write('<script src="./main.js?t=' + ts + '"><\/script>')
     </script>
+    <script language="javascript" type="text/javascript">
+        $(window).load(function() {
+            $('#loading').hide();
```
Comment (joerx) [inline schedule/index.html:135]: I'm not sure what this is doing, but it should be hooked in to the SessionStores 'data' event (see `main.js`, ca. line 257) - i.e. loader should display until the session data was loaded via ajax.

**Comment by joerx** [comment]
Can we please use a simpler icon, e.g. a black circle ball via http://www.ajaxload.info/?

**Comment by mariobehling** [comment]
@SilentFlame This is a nice thing to have, but we also have some basic features, we need. Would be great, if you could help with this one: https://github.com/fossasia/2016.fossasia.org/issues/28

**Comment by jigyasa-grover** [comment]
@SilentFlame There are merge conflicts with this PR. 

Also as a general tip, always add a preview link (viz. `<username>.github.io/2016.fossasia.org`) for an easier review.

**Comment by SilentFlame** [comment]
@jig08  are merge conflicts arise when we make a pull request from a branch which is non-existing on the main repo, or is it something else?

**Comment by joerx** [comment]
It's because we pushed conflicting changes to `gh-pages` after you forked. Please rebase your branch (https://git-scm.com/book/en/v2/Git-Branching-Rebasing)

**Comment by SilentFlame** [comment]
I have rebase my develop branch with the gh-pages on my local system then pushed the local gh-pages to the forked repo. now should it work? 
 @joerx can you have a look as does it still have merge conflicts?
so that I can proceed with my work here.

**Comment by jigyasa-grover** [comment]
@SilentFlame It does have.

**Comment by SilentFlame** [comment]
@jig08  unable to resolve can you help please?

**Comment by jigyasa-grover** [comment]
Go through the simple steps in [this guide](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md), it might help.

**Comment by joerx** [comment]
@SilentFlame did you pull the upstream `gh-pages` before rebasing?

**Comment by SilentFlame** [comment]
@joerx no I guess I don't, but currently I'm going through steps provided by @jig08.
let me try if it works.

**Pull request closed without merging**